images/build/Dockerfiles/Dockerfile

```diff
@@ -110,5 +112,6 @@ RUN set -ex \
     && cd /opt/php \
     && ln -s 5.6 5 \
     && echo "full" > /opt/oryx/.imagetype \
-    && echo "DEBIAN|${DEBIAN_FLAVOR}" | tr '[a-z]' '[A-Z]' > /opt/oryx/.ostype
+    && echo "DEBIAN|${DEBIAN_FLAVOR}" | tr '[a-z]' '[A-Z]' > /opt/oryx/.ostype \
+    && export ORYX_SDK_STORAGE_ACCOUNT_ACCESS_TOKEN=""
```

Comment:
Was just curious if we are going to go ahead and try the alternative that Paul mentioned so we can get rid of this 
`export ORYX_SDK_STORAGE_ACCOUNT_ACCESS_TOKEN=""` here?

Reply:
Feel free to watch the recording from yesterday for the more in depth explanation, but at a high level the changes I suggested are:
1. Update the `ORYX_SDK_STORAGE_ACCOUNT_ACCESS_TOKEN` env var in the Dockerfile `RUN` scripts to be `ORYX_SDK_STORAGE_ACCOUNT_ACCESS_TOKEN_PATH`. Update their value to be the path to the secret file instead of the token itself.
2. Update the scripts that use `ORYX_SDK_STORAGE_ACCOUNT_ACCESS_TOKEN` to instead pull the token out the path env var into a temporary variable (or directly into the url)
3. remove the `&& export ORYX_SDK_STORAGE_ACCOUNT_ACCESS_TOKEN=""` lines from the dockerfile, since we no longer save the token to an env var

Reply:
@waliMSFT - re-opening this conversation, as I cannot find these changes in the code